cmd/influxd/launcher/launcher.go

```diff
@@ -554,7 +554,7 @@ func (m *Launcher) run(ctx context.Context) (err error) {
 		// define the executor and build analytical storage middleware
 		combinedTaskService := taskbackend.NewAnalyticalStorage(m.logger.With(zap.String("service", "task-analytical-store")), m.kvService, m.kvService, pointsWriter, query.QueryServiceBridge{AsyncQueryService: m.queryController})
 		executor := taskexecutor.NewAsyncQueryServiceExecutor(m.logger.With(zap.String("service", "task-executor")), m.queryController, authSvc, combinedTaskService)
-
+		// TODO: check for env variable to decide whether to use old scheduler system or new
```

Comment:
I will create an issue to unify the OSS and IDPE initialization / setup. We'll want to do this before we integrate the new scheduler, so this TODO will probably be different / redundant.